scripts/what-the-fips/fix.js

```diff
@@ -7,7 +7,7 @@ const promisify = require('util').promisify;
 const csvParse = promisify(require('csv-parse'));
 
 const POPULATION_CSV_URL =
-  'https://raw.githubusercontent.com/covid-projections/covid-data-model/main/libs/datasets/sources/fips_population.csv';
+  'https://raw.githubusercontent.com/act-now-coalition/covid-data-model/main/data/misc/fips_population.csv';
```

Comment:
Thanks for fixing this. Though for context, this was a one-time script to debug some issues that we could probably delete if we wanted. 🤷

Reply:
Shall we just delete it then?

Reply:
Feel free!